src/get-started/java-binding.md

```diff
@@ -6,31 +6,20 @@ and Java services runtime environment.
 ## Installation
 
 To run a node with your Java service you need to use Exonum Java application.
-You can download an archive containing the application and all the necessary
-dependencies on [the Releases page][github-releases] on GitHub. We suggest using
-`debug` version during development and `release` for deployment.
 
-### Mac OS
+There are several installation options:
 
-For Mac users, we provide a [Homebrew][homebrew] repository, which gives the
-easiest way of installing Exonum Java App:
-
-```bash
-brew tap exonum/exonum
-brew install exonum-java
-```
-
-This will install `exonum-java` binary with all the necessary dependencies.
-However, you still need to install [Maven 3][maven-install] and follow the
-steps mentioned in [After Install][#after-install] section below.
-
-You can also install Exonum Java App manually:
+- [Manual installation](#manual-installation) - available on Mac OS and Linux
+- [Homebrew package](#homebrew-package) - available on Mac OS only, recommended for
+  Mac users
+- [Build from source](#build-from-source) - available on Mac OS and Linux, does not
+  recommended for beginners
```

Comment:
I think "not recommended for beginners" is redundant.

Reply:
Anyway, if you decide to keep this phrase, please edit it as follows: "is recommended only for advanced users"